app/services/set_reject_by_default.rb

```diff
@@ -28,11 +28,11 @@ def call
 
 private
 
-  def beyond_eoc?(date)
-    date >= CycleTimetable.find_closes
+  def beyond_reject_by_default?(date)
```

Comment:
Maybe give this a name to clarify it's about EOC? `beyond_end_of_cycle_reject_by_default_deadline?` or sth?